### ISSUE TITLE
Program crashed with Cannot create stack Roslyn error

### ISSUE THREAD

**Issue by snaulX** [mellinoe/veldrid#389]
When I run my program, window is created but it's empty and without title. Next I get messagebox with text like this "Cannot create stack Roslyn". Next my computer lags very hard until I'm not closing the program. I don't know what can cause this error but sources is open and you can see my code here: https://github.com/wings-studio/Enigma.Graphics/blob/develop/Enigma.Graphics.Sample/Program.cs  
What causes this issue and how I can fix it? My OS: Windows 10, 64-bit, graphics backend - Direct11

**Comment by TechnologicalPizza** [comment]
Downloading the zip of Enigma.Graphics seems to be missing some Enigma.Input assembly reference.

**Comment by snaulX** [comment]
Looks strange because I removed Enigma.Input from references and commit it. Just remove it I think code must compiles.

**Comment by TechnologicalPizza** [comment]
Your game loop (in Program.cs) only calls rendering methods.
You never call renderer.GraphicsDevice.**SwapBuffers**();
And as you are not calling `window.Update(deltaSeconds);`, the window is also unresponsive.
And `Scene.Add` also tries to create device resources without the CommandList ever being initialized/begun, so there is a bit of redesign to be done.

![image](https://user-images.githubusercontent.com/23627133/120075282-f1dc9e80-c0a0-11eb-8fe7-6926252ebb96.png)
This is what I changed to get a working window (also changed clear color to Red to verify). I also commented out Scene.Add for now because it's not implemented correctly as mentioned.

Also noticed that your Scene.UpdateResources method allocates a new CommandList every time, try to cache it like you do for drawing. 
And to nit-pick more; the IRenderable interface seems a bit too unsafe. I would recommend passing in GraphicsDevice and CommandList as method parameters instead. And
